server/channels/app/email/email.go

```diff
@@ -496,6 +496,7 @@ func (es *Service) SendGuestInviteEmails(
 	errorWhenNotSent bool,
 	isSystemAdmin bool,
 	isFirstAdmin bool,
+	isEasyLogin bool,
```

Comment:
Not for this PR, we might just gather all these options into something like `GuestInviteParams` or something like that. It's barely readable 🤷